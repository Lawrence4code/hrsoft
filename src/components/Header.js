import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Hrsoft</h1>
    <NavLink to="/" activeClassName="isActive" exact>
      Dashboard
    </NavLink>
    <NavLink to="/create" activeClassName="isActive">
      Add Client
    </NavLink>
  </header>
);

export default Header;
