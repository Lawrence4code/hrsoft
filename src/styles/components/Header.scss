.header {
  background: $dark-blue;
  padding: 1rem 0;
  &__content {
    width: 90%;
    margin: 0 auto;
    display: flex;
    justify-content: space-between;
    align-items: center;
  }

  &__logo {
    text-decoration: none;
    color: $white;

    &--special {
      background: $white;
      color: $dark-blue;
      border-radius: 2px;
      padding: 2px;
      margin-right: 3px;
    }
  }

  &__button {
    color: $white;
    background: transparent;
    cursor: pointer;
    transition: all 0.5s ease;
    font-size: 12px;
    border: none;
    font-size: 0.9rem;

    &:hover {
      color: $red;
    }
  }
}
