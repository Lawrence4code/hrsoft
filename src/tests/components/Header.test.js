import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render Header correclty', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
  // expect(wrapper.find('h1').text()).toBe('Hrsoft');
  // expect(wrapper.find('h1').length).toBe(2);
});
