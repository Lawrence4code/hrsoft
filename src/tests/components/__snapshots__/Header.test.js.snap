// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correclty 1`] = `
<header>
  <h1>
    Hrsoft
  </h1>
  <NavLink
    activeClassName="isActive"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="isActive"
    aria-current="page"
    to="/create"
  >
    Add Client
  </NavLink>
</header>
`;
